package.json

```diff
@@ -480,7 +476,12 @@
         "titanium.project.defaultI18nLanguage": {
           "type": "string",
           "default": "en",
-          "description": "%titanium.config.project.defaultI18nLanguage%"
+          "description": "Default language to use for i18n autocomplete."
+        },
+        "titanium.ios.organizations": {
+          "type": "string",
```

Comment:
Is it possible to use `object` instead of `string` here to allow folks to do a key/value as configuration? That might simplify how we look up the organization also

```
"titanium.ios.organizations": {
	"certId1": "Acme"
}
```